patterns/newtype.md

```diff
@@ -1,8 +1,11 @@
 # Newtype
 
-Rust has strong static types. This can be very different than what you are used to if you are coming from a loosely-typed language. Don't worry, though. Once you get used to them, you'll find the types actually make your life easier. Why? Because you are making implicit assumptions explicit.
+What if in some cases we want a type to behave similar to another type but
+still different? What if we want to enforce some behavior at compile time
+which using only type aliases is not enough? For example, have a custom
+`Display` implementation for `String` for security reasons (password).
 
-A really convenient application of the Rust type system is the Newtype pattern.
+We could use the newtype pattern to provide type safety and encapsulation.
```

Comment:
```suggestion
For aforementioned and such cases we could use the `Newtype` pattern to provide __type safety and encapsulation__.
```

Reply:
```suggestion
For such cases we could use the `Newtype` pattern to provide __type safety__ and __encapsulation__.
```

aforementioned is a hard word, I think it is better to remove that.